src/azure-cli/HISTORY.rst

```diff
@@ -11,6 +11,14 @@ Release History
 
 * az webapp up detects env folder and removes it from compressed file used for deployment
 
+**Batch**
+
+* Expanded `--json-file` capabilities of `az batch pool create` to allow for specifying MountConfigurations for file system mounts(see https://docs.microsoft.com/en-us/rest/api/batchservice/pool/add#request-body for structure)
```

Comment:
Somewhat confusing that you phrased these two things differently (one mentions its an optional property whereas MountConfigurations doesn't)

Reply:
minor: wrap `publicIPs`/`MountConfigurations` in backticks for markdown, and also case them consistently.